addons/binding/org.openhab.binding.lutron/README.md

```diff
@@ -8,70 +8,175 @@ This binding integrates with [Lutron](http://www.lutron.com) light control syste
+* ttkeypad - Tabletop seeTouch Keypad
+* pico - Pico Keypad
+* vcrx - Visor control receiver
+* virtualkeypad - Repeater virtual keypad
 
 ## Discovery
 
 Discovery is supported for RadioRA 2.
 Discovered RadioRA 2 main repeaters will use the default lutron/integration credentials.
 This can be changed in the main repeater thing configuration.
 
-**Note:** discovery of devices paired with a bridge should work on systems other than Radio RA 2; however, the bridge itself will need to be manually added as bridge discovery is only supported for Radio RA 2.
+**Note:** discovery of devices paired with a bridge should work on systems other than RadioRA 2; however, the bridge itself will need to be manually added as bridge discovery is only supported for RadioRA 2.
 
 ## Binding Configuration
 
 This binding does not require any special configuration.
 
-## Thing Configuration
+## Thing Configuration and Usage
+
+Each Lutron thing requires the integration ID of the corresponding item in the Lutron system. The integration IDs can be retrieved from the integration report generated by the Lutron software.
 
-The bridge requires the IP address of the bridge as well as the telnet username and password to log in to the bridge.
+### Bridge
+
+The bridge may currently be a RadioRA 2 main repeater or a Caseta Smart Bridge Pro. The bridge configuration requires the IP address of the bridge as well as the telnet username and password to log in to the bridge.
 
 ```
 Thing lutron:ipbridge:radiora2 [ ipAddress="192.168.1.2", user="lutron", password="integration" ]
 ```
 
-Each Lutron thing requires the integration ID of the corresponding item in the Lutron system.
-The integration IDs can be retrieved from the integration report generated by the Lutron software.
-Dimmers can optionally be configured to specify a fade in and fade out time in seconds.
-In the thing file, this looks e.g. like
+### Dimmers
+
+Dimmers can optionally be configured to specify a fade in and fade out time in seconds using the fadeInTime and fadeOutTime parameters. In the thing file, this looks e.g. like:
 
 ```
 Thing lutron:dimmer:livingroom (lutron:ipbridge:radiora2) [ integrationId=8, fadeInTime=0.5, fadeOutTime=5 ]
 ```
 
+### Switches
+
+Switches take no additional parameters besides integrationId.
+
+Example:
+
+```
+Thing lutron:switch:porch (lutron:ipbridge:radiora2) [ integrationId=8 ]
+```
+
+### CCO Modules
+
+Cold contact output (cco) things accept outputType and defaultPulse parameters. The outputType parameter is a string that should be set to "Pulsed" for pulsed CCOs or "Maintained" for non-pulsed CCOs. The default is "Pulsed", since this is generally the safer wrong setting. The defaultPulse parameter sets the pulse length in seconds for a pulsed output.  It can range from 0.1 to 100 and defaults to 0.5. It is ignored if outputType="Maintained".  Be aware that the Lutron controller may round the pulse length to the nearest 0.25 seconds.
```

Comment:
Most of the literature I've seen refers to CCI/CCOs as contact closure input/outputs.

Reply:
Yes, "contact closure" is really the proper terminology.  I've changed it everywhere.